### PR TITLE
Gyra_Solune's unit description updates

### PR DESCRIPTION
**Pull request by ln-zookeeper**
I'm putting these in PR form to make it easier for more people to review them.

Note: don't merge. I can merge this myself after I've pushed all the ones I intend to and they've been reviewed.

Source: https://forums.wesnoth.org/viewtopic.php?f=32&t=43721

### CODE REVIEW

**Comment by Wedge009** [comment]
I think these are all well-written with no major issues with spelling or grammar that I can see.

The only change I'm a bit uncomfortable with is the end of the Merman Warrior's new description: '...as any who try to match them on their homes can attest.'

I wonder if this should be '_in_ their homes' or '_in_ their home waters'. Minor gripe, but that's all I have against this. Otherwise, merge away.

**Comment by Espreon** [comment]
For the most part, these seem fine, but there are some issues I must point out:

1. This seems to introduce some of singular verbs being used with coordinated subjects in a way that strikes me as strange. For example: "Their resilience and skill with their iconic tridents makes"  rather than "Their resilience and skill with their iconic tridents make", but in things like "their diligent bearing and basic insight of combat means", it seems fine and perhaps better than with a verb conjugated in the plural, strangely.

2. The Sergeant's description starts with the singular and then switches to the plural. I would completely change it to the plural: "Sergeants are low-ranking officers of human armies"

3. The Orcish Warlord's and perhaps the Grunt's descriptions seem to string things together with commas inappropriately. For example, in "Wearing the most ornate armor they plunder from their enemies, the Orcish Warlords are true champions of battle, their mastery of the blade able to match even the finest from vaunted elvish and human schools of swordsmen.", I would either make "their mastery ..." a new sentence or connect it to the previous one with a semicolon rather than a comma.

4. I'm not so sure if it is proper to have a comma before "and" that introduces a non-independent clause. My intuition is to write "Lieutenants are some of the more respected commanding officers among humans and are often seen" rather than "Lieutenants are some of the more respected commanding officers among humans, and are often seen". I would treat "but" the same way, but it does not seem as bad as it does with "and", for some reason.

5. In the Peasant's description, ", though, and ..." looks silly, unfortunately. At the very least, I would change the comma before "and" to a semicolon or drop "and" and make the comma a semicolon. So: ", though; and ..." or ", though; ...".

Good work!

**Comment by Wedge009** [comment]
Darn, I saw your comments on that in IRC (at least point 1), but missed it completely when reading it here. Definitely agreed with your assessment on IRC, but I suppose it goes to show how easy it is to miss when reading as part of a larger block of text.

File: data/core/units/orcs/Warlord.cfg [hunk 1]
```diff
@@ -15,7 +15,7 @@
     {AMLA_DEFAULT}
     cost=48
     usage=fighter
-    description= _ "Only orcs with both great cunning and unsurpassed strength can become Warlords. Masters of the sword, and even possessing some skill with the bow, these beast-warriors lead their brethren with iron will and iron hand."
+    description= _ "In a race where might is the be-all end-all of supremacy and respect, the orcs who achieve victory time and time again become the rulers of much greater orcish tribes. Wearing the most ornate armor they plunder from their enemies, the Orcish Warlords are true champions of battle, their mastery of the blade able to match even the finest from vaunted elvish and human schools of swordsmen. They even have practice with the bow, their usage of this ordinarily cowardly weapon offset by the sheer admiration their warriors have for them. However, the Warlords lack the insight for strategy and leading charisma of some of the more cunning orcs, and so the hordes they lead often fall simply due to their inability to truly lead."
```
Comment (GregoryLundberg) [inline data/core/units/orcs/Warlord.cfg]: ... their mastery of the blade **makes them** able to match ...

File: data/core/units/orcs/Warrior.cfg [hunk 2]
```diff
@@ -14,7 +14,7 @@
     advances_to=Orcish Warlord
     cost=26
     usage=fighter
-    description= _ "Having mastered the art of dual-sword fighting, Orcish Warriors are highly respected among the Orcs, and highly feared by their foes. Their only weakness is their inability to wield the bow."
+    description= _ "Orcish Warriors are those orcs who have proved their strength on the battlefield, choosing to display their skill by fighting with a sword in each hand. Their prowess in battle has gained them the respect of other orcs, and they will typically be seen ordering lesser bands of raiders. Despite their position, they rarely possess the commanding skills held by the leaders of other armies."
```
Comment (GregoryLundberg) [inline data/core/units/orcs/Warrior.cfg]: .. and they **typically will** be seen ..

or

.. and**, typically,** they will be seen ..

File: data/core/units/merfolk/Triton.cfg [hunk 3]
```diff
@@ -16,7 +16,7 @@
     cost=43
     usage=fighter
     # wmllint: local spelling tritons
-    description= _ "Tritons are combat masters of the sea. Skilled in use of the trident, Tritons easily defeat any enemy foolish enough to wander into their preferred environment."
+    description= _ "Tritons are combat masters of the sea. Skilled in use of the trident, Tritons easily defeat any enemy foolish enough to wander into their preferred environment. Their mastery of their weapon affords them flexibility in its use, and the highest-ranking among them use weaponry more comparable to halberds than simple spears."
```
Comment (GregoryLundberg) [inline data/core/units/merfolk/Triton.cfg]: ... are **the** combat masters ...

File: data/core/units/humans/Peasant.cfg [hunk 4]
```diff
@@ -15,7 +15,7 @@
     advances_to=Spearman,Bowman
     cost=8
     usage=fighter
-    description= _ "Peasants are the backbone of the rural economy, and the soldiers of last resort. While not warlike by nature, they will stubbornly defend their homes. However, if you’re hurling peasants at your foes, you’re clearly out of superior forces."
+    description= _ "Peasants are the backbone of the rural economy, and the soldiers of last resort. While not warlike by nature, they will stubbornly defend their homes. Their lack of training and discipline makes them unfit to hold a line, though; their reliance on farming tools as weaponry means they are only on the field of battle under the most desperate situations."
```
Comment (GregoryLundberg) [inline data/core/units/humans/Peasant.cfg]: .. unfit to hold a line, **and** their reliance ..

or

.. unfit to hold a line**. Their** reliance ..

File: data/core/units/orcs/Warrior.cfg (hunk 2 above)
Comment (CelticMinstrel) [inline data/core/units/orcs/Warrior.cfg]: For me, all three versions work.

**Pull request merged**